docling/backend/msword_backend.py

```diff
@@ -934,13 +955,14 @@ def _handle_text_elements(  # noqa: C901
                 paragraph_elements=paragraph_elements,
             )
             for text, format, hyperlink in paragraph_elements:
-                doc.add_text(
+                t2 = doc.add_text(
                     label=DocItemLabel.PARAGRAPH,
```

Comment:
Can we use `TEXT` instead of `PARAGHRAPH`. I would like to deprecate `PARAGHRAPH`.